doc/Language/traps.pod6

```diff
@@ -797,7 +797,7 @@ for other, more common, things:
 
 The C<^>, C<|>, and C<&> are I<not> bitwise operators, they create
 L<Junctions|/type/Junction>. The corresponding bitwise operators in Raku are:
-C<+^>, C<+|>, C<+&> for integers and C<?^>, C<?|>, C<?&> for booleans.
+C<+^>, C<+|>, C<+&> for integers and C<?^>, C<?|>, C<?&> for Booleans.
 
```

Comment:
This is one case where `Bool` would suit better than Booleans, since you're referring to specific types. Anyway, that was not in the original, so capitalization is OK (but a fix further down the line would be better)